shelltests/wintest.ps1

```diff
@@ -46,7 +33,7 @@ Pop-Location
 
 # Function to run rebar3
 function Rebar() {
```

Comment:
I guess this could be simplified further (relying on the action's `rebar3` instead of cloning `master`, etc., but I didn't want to mess with this script too much. Let me know if you prefer to further simplify it.